### PR TITLE
Bug fixes for parceler, parse user and some toolbar enhancement

### PR DESCRIPTION
**Pull request by DeepakSumo**
- Bug fix for user not getting pulled  #26 
- Workaround for parcelebale to work with parse objects 
- App name change, app icon, app logo (lets not stick with fashion in particular but anything renting service) #24

### CODE REVIEW

**Comment by DeepakSumo** [comment]
@deborshisaha @rakheegupta 
Please review, merge and pull

**Comment by deborshisaha** [comment]
@DeepakSumo Can you please paste a screenshot of the toolbar?

File: Android/app/src/main/java/fashiome/android/adapters/ProductAdapter.java [hunk 1]
```diff
@@ -213,9 +213,9 @@ public void configure(Product product) {
             tvProductTitle.setText(product.getProductName());
             tvProductDescription.setText(product.getProductDescription());
 
-//            if ( product.getProductPostedBy().getFullName() != null) {
-//                tvProductByUserName.setText( product.getProductPostedBy().getFullName());
-//            }
+            if (product.getProductPostedBy().getUsername() != null) {
+                tvProductByUserName.setText(product.getProductPostedBy().getUsername());
```
Comment (deborshisaha) [inline Android/app/src/main/java/fashiome/android/adapters/ProductAdapter.java]: getUsername is easy. :) I am interested in the full name.

File: Android/app/src/main/java/fashiome/android/adapters/ProductPagerAdapter.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public Object instantiateItem(ViewGroup container, int position) {
 
         final ImageView imageView = (ImageView) itemView.findViewById(R.id.imageView);
 
-        String URLString = ImageURLGenerator.getInstance(this.mContext).URLForImageWithCloudinaryPublicId(mProduct.getImageCloudinaryPublicId(position), Utils.getScreenWidth(this.mContext));
+        String URLString = ImageURLGenerator.getInstance(this.mContext).URLForImageWithCloudinaryPublicId(mProduct.getImageCloudinaryPublicId(position), Utils.getScreenWithInDp(this.mContext));
```
Comment (deborshisaha) [inline Android/app/src/main/java/fashiome/android/adapters/ProductPagerAdapter.java]: Utils.getScreenWidthInDp  - spelling

File: Android/app/src/main/java/fashiome/android/activities/HomeActivity.java [hunk 3]
```diff
@@ -98,7 +92,31 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         switch (requestCode) {
 
             case 100:
-                mProductsFragment.addNewProductToList((Product) data.getParcelableExtra("product"));
+
```
Comment (deborshisaha) [inline Android/app/src/main/java/fashiome/android/activities/HomeActivity.java]: Adding the fragment to the list is not correct! It is not a timeline that the feed will remain constant. Based on search result or where the user is (geo location) the result set will change.

File: Android/app/src/main/java/fashiome/android/adapters/ProductAdapter.java (hunk 1 above)
Comment (DeepakSumo) [inline Android/app/src/main/java/fashiome/android/adapters/ProductAdapter.java]: You can anything now .. Problem is from FB we get fullname we store as username only

File: Android/app/src/main/java/fashiome/android/adapters/ProductPagerAdapter.java (hunk 2 above)
Comment (DeepakSumo) [inline Android/app/src/main/java/fashiome/android/adapters/ProductPagerAdapter.java]: nice catch :)

**Pull request merged**